lib/urls.ts

```diff
@@ -4,5 +4,7 @@ export const EthHomepageURLs = {
   eth_holders: "https://api.growthepie.xyz/v1/eim/eth_holders.json",
 };
 
-export const DiscordURL = "https://discord.gg/yyH8NjJkkM";
+export const DiscordURL = "https://discord.gg/dAM6jwm9Ww";
```

Comment:
```suggestion
export const DiscordURL = "https://discord.com/invite/sdCzcnMHVb";
```
this url won't expire